### PR TITLE
Appveyor CI - DO NOT MERGE (still)

### PR DESCRIPTION
**Pull request by klokane**
Testing enviroment for build in appveyor.
Will ping when be ready to merge

### CODE REVIEW

**Comment by pksunkara** [comment]
I have updated this PR. This is failing on windows.

**Comment by klokane** [comment]
It is based on older master, will rebase and should be ok

**Comment by pksunkara** [comment]
I had already rebased it and it is still failing.

Pavan Kumar Sunkara

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 1.0.{build}
+
+build_script:
+  - git submodule update --init --recursive
+  - vcbuild.bat MSVC2013
+
+test_script:
+  - npm install -g cucumber
```
Comment (pksunkara) [inline appveyor.yml]: This is probably the problem with appveyor. I will look into this.

**Pull request merged**